Formula/arcs.rb

```diff
@@ -40,9 +41,17 @@ def install
       "--prefix=#{prefix}",
       "--with-boost=#{Formula["boost"].opt_include}"
     system "make", "install"
+    libexec_src = Pathname.new("#{libexec}/bin/src")
+    libexec_src.install "src/long-to-linked-pe"
+    libexec_bin = Pathname.new("#{libexec}/bin/Examples")
+    libexec_bin.install "Examples/makeTSVfile.py"
+    libexec_bin.install "Examples/arcs-make"
+    (bin/"arcs-make").write_env_script libexec/"bin/Examples/arcs-make", PYTHONPATH: ENV["PYTHONPATH"]
   end
 
   test do
     assert_match "Usage", shell_output("#{bin}/arcs --help")
+    assert_match "Usage", shell_output("#{bin}/long-to-linked-pe --help 2>&1")
```

Comment:
I don't see in the formula where `#{bin}/long-to-linked-pe` is installed. Is it installed by `make install`? The formula installs it to `libexec/bin/src` in `libexec_src.install "src/long-to-linked-pe"`, but this test passes, so it must also be installed to `bin`. It ought to be named with an `arcs-` prefix if it's being installed to `bin`, such as `arcs-long-to-linked-pe`.

Reply:
Yes, it is installed by `make install`. I can also make a note of your suggestion for naming for a future release - I wouldn't need it to necessarily be installed to `bin`, but I didn't see how to avoid that since it's installed by `make install`?

Reply:
Adding an `arcs-` prefix to and installing it to `bin` is easiest (rather than `libexec`).